main.c

```diff
@@ -99,6 +99,23 @@ void powerEnterSleepMode(void)
   // APP_ERROR_CHECK(err_code);
   nrf_gpio_cfg_sense_set(BUTTON_PIN, NRF_GPIO_PIN_SENSE_LOW);
 
+  // LEDs
+  nrf_gpio_cfg_input(LED1_PIN, NRF_GPIO_PIN_PULLUP);

```

Comment:
It looks like this is actually hooked up to VDDA. If that gets powered down, then this should just be Hi_Z